### PR TITLE
Unify test names

### PR DESCRIPTION
**Pull request by sparrowDom**
Unifies test names to: `TEST_NAME.NETWORK_NAME.fork.js`

### CODE REVIEW

File: contracts/test/_global-hooks.js [hunk 1]
```diff
@@ -34,11 +39,25 @@ mocha.before(function () {
...
+      return isArbTestFile;
+    } else if (isMainnetForkTest) {
+      return isMainnetForkTestFile && !isArbTestFile;
+    } else if (isBaseFork) {
+      return isBaseTestFile;
+    } else if (isHoleskyFork) {
+      return isHoleskyTestFile;
+    } else {
+      // else is unit test
+      return !isMainnetForkTestFile && !isHoleskyTestFile;
```
Comment (PraneshASP) [inline contracts/test/_global-hooks.js:58]: does it require `!isArbTestFile && !isBaseTestFile`?

Comment (sparrowDom) [reply contracts/test/_global-hooks.js]: great catch! corrected here: https://github.com/OriginProtocol/origin-dollar/pull/2094/commits/877cc11c84fe9019c0b2f6cfd3e50814177aa582

Comment (PraneshASP) [reply contracts/test/_global-hooks.js]: much cleaner, thanks!

**Review by PraneshASP — APPROVED** [review]
LGTM

File: contracts/test/_global-hooks.js [hunk 2]
```diff
@@ -34,11 +39,26 @@ mocha.before(function () {
...
-  root.suites = root.suites.filter(
-    (s) =>
-      s.file.endsWith(".fork-test.js") == isForkTest &&
-      s.file.endsWith(".arb.fork-test.js") == isArbFork
-  );
+  root.suites = root.suites.filter((s) => {
+    const isMainnetForkTestFile = s.file.endsWith("mainnet.fork-test.js");
+    const isHoleskyTestFile = s.file.endsWith(".holesky.fork-test.js");
+    const isArbTestFile = s.file.endsWith(".arb.fork-test.js");
+    const isBaseTestFile = s.file.endsWith(".base.fork-test.js");
```
Comment (shahthepro) [inline contracts/test/_global-hooks.js:46]: One of the reasons I suggested this pattern is that we can parse the network from the file name and check it against `FORK_NETWORK_NAME` to simplify this a lot

File: contracts/test/_global-hooks.js [hunk 3]
```diff
@@ -34,11 +39,26 @@ mocha.before(function () {
...
+    const unitTest = !s.file.endsWith(".fork-test.js");
```
Comment (shahthepro) [inline contracts/test/_global-hooks.js:47]: Should this be `.fork.js` and not `.fork-test.js`?

**Pull request merged**